src/components/pages/login/LoginPage.js

```diff
@@ -1,8 +1,36 @@
-
-import LoginForm from "./LoginForm";
+import styled from "styled-components"
+import LoginForm from "./LoginForm"
+import Logo from "./../../reusable-ui/Logo"
+import burgerBackground from "./../../../assets/images/F03-burger-background.jpg"
 
 export default function LoginPage() {
-
   // Affichage
-  return <LoginForm />
+  return (
+    <LoginPageWrapper>
```

Comment:
```suggestion
    <LoginPageStyled>
```

Pas la peine de préciser Wrapper, on le sait déjà. En revanche, ce qu'on ne sait pas, c'est que c'est en fait un Styled-Component. Le bon usage qu'on retrouve en entreprise c'est de rajouter "Styled" ici.

Par ailleurs, "LoginPageWrapper" laisse penser que c'est un composant à part entière qu'on peut retrouver dans components/reusable-ui alors que ce n'est pas le cas. C'est uniquement un Styled-Component.